Cargo.toml

```diff
@@ -1,17 +1,21 @@
 [package]
 name         = "cryptobox-c"
-version      = "1.0.0"
+version      = "2.0.0"
 authors      = ["Wire Swiss GmbH <support@wire.com>"]
 description  = "C-FFI to cryptobox"
 homepage     = "https://github.com/wireapp/cryptobox-c"
 repository   = "git@github.com:wireapp/cryptobox-c.git"
 license      = "GPL-3.0"
 
 [lib]
-name         = "cryptobox"
-crate-type   = ["cdylib"]
+name         = "cryptoboxdb"
+crate-type   = ["staticlib","cdylib"]
 
 [dependencies]
-cryptobox = { git = "https://github.com/wireapp/cryptobox", branch = "develop" }
+cryptobox = { git = "https://github.com/zhaohansprt/cryptobox", branch = "crdb" }
```

Comment:
This should refer to wireapp/cryptobox now (the cockroachdb-experimental branch)

Reply:
yes it should be,my overlooked